packages/babel-template/src/formatters.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+export type Formatter<T> = {
+  code: string => string,
+  validate: BabelNodeFile => void,
+  unwrap: BabelNodeFile => T,
+};
+
+function makeStatementFormatter<T>(
+  fn: (Array<BabelNodeStatement>) => T,
+): Formatter<T> {
+  return {
+    code: str => `"==@babel/template==";;\n${str}`,
```

Comment:
Curious for thoughts on this. Since we want to force things to be statements and not directives, the easiest thing is just to prepend a `;` to the string. I felt like it'd be a little weird for people to get errors like
```
SyntaxError: Unexpected token, expected ; (2:4)
  1 | ;
> 2 | omg 45
    |    ^
```
so I threw a little header in there so it's clearer, e.g.
```
SyntaxError: Unexpected token, expected ; (2:4)
  1 | "==@babel/template==";;
> 2 | omg 45
    |    ^
```

Reply:
this certainly gives at least some context